src/components/Banner/styled.js

```diff
@@ -17,3 +13,7 @@ export const BannerContainer = styled(Container)`
     background-color: #222222;
   }
 `;
+
+export const Img = styled.img`
+  width: 100%;
+`;
```

Comment:
Perhaps a more explicit name would be better. What do you think about `FullWidthImage` or `FluidImage`? Something that lets you know that this image will fill the width to 100%.